src/main/scala/outwatch/dom/Compat.scala

```diff
@@ -27,4 +27,25 @@ trait Handlers {
   def createHandler[T](defaultValues: T*): IO[Pipe[T, T]] = Handler.create[T](defaultValues: _*)
 }
 
-object Handlers extends Handlers
\ No newline at end of file
+object Handlers extends Handlers
+
+
+trait Attributes extends
+  DomAttrs with
+  DomReflectedAttrs with
+  DomProps with
+  DomEvents with
+  OutwatchAttributes
+
+@deprecated("Attributes is deprecated, use one of DomAttrs, DomReflectedAttrs,  DomProps, DomEvents or OutwatchAttributes", "0.11.0")
+object Attributes extends Attributes
+
+
+trait DomAttrsCompat[RA[_, _]] { self: ReflectedAttrs[RA] =>
+  lazy val `class`: RA[String, String] = className
+
+  lazy val `for`: RA[String, String] = forId
+}
+
+trait Tags extends DomTags with DomTagsExtra
```

Comment:
We should also deprecate `Tags` here, right?

Reply:
I didn't see any reason to deprecate it. What do you think about renaming all `DomTags`, `DomTagsExtra`, `DomProps`, `DomEvents`, ... to `Tags`, `TagsExtra`, `Props`, `Events`...

They are already in the dom package.

Reply:
Good idea!